<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QueryCodeClass</class>
 <widget class="QWidget" name="QueryCodeClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QueryCode</string>
  </property>
  <widget class="QTreeWidget" name="treeWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>561</width>
     <height>291</height>
    </rect>
   </property>
   <attribute name="headerVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="headerStretchLastSection">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>1</string>
    </property>
   </column>
   <item>
    <property name="text">
     <string>22</string>
    </property>
    <item>
     <property name="text">
      <string>2</string>
     </property>
     <item>
      <property name="text">
       <string>3</string>
      </property>
      <item>
       <property name="text">
        <string>31</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>32</string>
       </property>
      </item>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>33</string>
      </property>
     </item>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>50</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>当前代号：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Code">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>50</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>输入查询：</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="querycode.qrc"/>
 </resources>
 <connections/>
</ui>
